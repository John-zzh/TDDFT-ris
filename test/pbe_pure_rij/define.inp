

a coord
ired
*
b all def2-TZVP
*
eht
y
0
y

dft
on
func
pbe
grid
3

ri
on

ex
rpas
q
a 5
*
q

q

