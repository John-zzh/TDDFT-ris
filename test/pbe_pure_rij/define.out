
 define (compt215) : TURBOMOLE rev. V7-6-2381-g96b3af1 compiled 16 Jan 2023 at 13:48:01
 Copyright (C) 2023 TURBOMOLE GmbH, Karlsruhe


    2023-03-28 01:48:11.459 

                    HOST NAME = compt215
             OPERATING SYSTEM = unix
   STANDARD BASIS SET LIBRARY = /home/smp201/dev/turbomole/basen/
  ALTERNATE BASIS SET LIBRARY = /home/smp201/dev/turbomole/basold/
 LIBRARY FOR RI-J  BASIS SETS = /home/smp201/dev/turbomole/jbasen/
 LIBRARY FOR RI-JK BASIS SETS = /home/smp201/dev/turbomole/jkbasen/
 LIBRARY FOR RIMP2/RICC2 SETS = /home/smp201/dev/turbomole/cbasen/
 LIBRARY FOR RIR12 BASIS SETS = /home/smp201/dev/turbomole/cabasen/
 LIBRARY FOR OEP   BASIS SETS = /home/smp201/dev/turbomole/xbasen/
            STRUCTURE LIBRARY = /home/smp201/dev/turbomole/structures/
 

 ***********************************************************
 *                                                         *
 *                       D E F I N E                       *
 *                                                         *
 *         TURBOMOLE'S  INTERACTIVE  INPUT  PROGRAM        *
 *                                                         *
 *  Quantum Chemistry Group       University of Karlsruhe  *
 *                                                         *
 ***********************************************************


 FILE control ALREADY EXISTS 
 I WILL PLUG IN THE NEW DATA.

 DATA WILL BE TAKEN FROM control BY DEFAULT
 
 INPUT TITLE 

 SYMMETRY c1   AND CARTESIAN COORDINATES FOR   6 ATOMS 
 HAVE BEEN READ FROM THE DEFAULT INPUT FILE control . 
 DEFINITIONS OF INTERNAL COORDINATES HAVE  N O T  BEEN READ. 
 SPECIFICATION OF BOND TOPOLOGY HAS  N O T  BEEN READ. 
 DO YOU WANT TO CHANGE THE GEOMETRY DATA ?  DEFAULT=n  GOBACK=&


 ATOMIC ATTRIBUTE DATA (BASES,CHARGES,MASSES,ECPS) HAVE BEEN
 TAKEN FROM THE DEFAULT INPUT FILE control.
 DO YOU WANT TO CHANGE THESE DATA ? DEFAULT=n  GOBACK=&


              +--------------------------------------------------+
              |               basis set information              |
              +--------------------------------------------------+

              we will work with the 1s 3p 5d 7f 9g ... basis set
              ...i.e. with spherical basis functions...

   type   atoms  prim   cont   basis
   ---------------------------------------------------------------------------
    c        1     46     31   def2-TZVP   [5s3p2d1f|11s6p2d1f]
    h        4      8      6   def2-TZVP   [3s1p|5s1p]
    o        1     46     31   def2-TZVP   [5s3p2d1f|11s6p2d1f]
   ---------------------------------------------------------------------------
   total:    6    124     86
   ---------------------------------------------------------------------------

   total number of primitive shells          :   46
   total number of contracted shells         :   38
   total number of cartesian basis functions :   96
   total number of SCF-basis functions       :   86


              ATOMIC COORDINATES              ATOM SHELLS CHARGE PSEUDO MASS
    -9.24314180    6.23174984    0.00054802    c     11      6.     0 12.011
    -9.98177905    5.77299993   -1.91166584    h      4      1.     0  1.008
    -6.60094563    6.20640861   -0.00619830    o     11      8.     0 15.999
    -9.98177905    4.88256098    1.43120298    h      4      1.     0  1.008
    -9.90212710    8.15492412    0.51283388    h      4      1.     0  1.008
    -6.10304059    4.45939461   -0.47154336    h      4      1.     0  1.008
 

 we will work with the  1s 3p 5d  7f

 there are 1 real representations :   a   

    mo occupation :
   irrep   mo's   occupied
    a       86        9
 
 number of basis functions   :    86
 number of occupied orbitals :     9
 

 reading orbital data $scfmo  from file mos
 orbital characterization : scfconv=7

 MOLECULAR ORBITAL DATA (OCCUPATION NUMBERS,MOS) HAVE BEEN 
 TAKEN FROM THE DEFAULT INPUT FILE control . 
 DO YOU WANT TO CHANGE THESE DATA ?  DEFAULT=n  GOBACK=&


 orbitals $scfmo will be written to file mos

 NO  'derivative'  SPECIFIC KEYWORDS / DATA GROUPS HAVE BEEN FOUND
 IN INPUT CONTROL FILE control


 NO <relax> SPECIFIC KEYWORDS / DATA GROUPS HAVE BEEN FOUND
 IN INPUT CONTROL FILE control


 PROVIDING FORCE RELAXATION DEFAULT PARAMETERS ...

 DO YOU WANT TO DELETE DATA GROUPS LIKE
     $energy
     $grad
     $dipole
 LEFT OVER FROM PREVIOUS CALCULATIONS ? DEFAULT(n)


 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


 delete data group $energy ? DEFAULT(y) 

 try to be more specific, please 


 delete data group $grad ? DEFAULT(y) 

 delete data group $dipole ? DEFAULT(y) 

 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


 try to be more specific, please 


    mo occupation :
   irrep   mo's   occupied
    a       86        9
 
 number of basis functions   :    86
 number of occupied orbitals :     9
 
 GENERAL MENU : SELECT YOUR TOPIC 
 scf    : SELECT NON-DEFAULT SCF PARAMETER 
 mp2    : OPTIONS AND DATA GROUPS FOR rimp2 and mpgrad
 cc     : OPTIONS AND DATA GROUPS FOR ricc2
 pnocc  : OPTIONS AND DATA GROUPS FOR pnoccsd
 ex     : EXCITED STATE AND RESPONSE OPTIONS
 prop   : SELECT TOOLS FOR SCF-ORBITAL ANALYSIS 
 drv    : SELECT NON-DEFAULT INPUT PARAMETER FOR EVALUATION
          OF ANALYTICAL ENERGY DERIVATIVES 
          (GRADIENTS, FORCE CONSTANTS) 
 rex    : SELECT OPTIONS FOR GEOMETRY UPDATES USING RELAX
 stp    : SELECT NON-DEFAULT STRUCTURE OPTIMIZATION PARAMETER
 e      : DEFINE EXTERNAL ELECTROSTATIC FIELD 
 dft    : DFT Parameters 
 ri     : RI Parameters 
 rijk   : RI-JK-HF Parameters 
 rirpa  : RIRPA Parameters 
 gw     : OPTIONS AND DATA GROUPS FOR GW (escf) 
 senex  : seminumeric exchange parameters 
 hybno  : hybrid Noga/Diag parameters
 dsp    : DFT dispersion correction
 nmr    : NMR shift parameters
 ncoup  : NMR coupling parameters
 trunc  : USE TRUNCATED AUXBASIS DURING ITERATIONS 
 marij  : MULTIPOLE ACCELERATED RI-J 
 fde    : Frozen Density Embedding 

 dis    : DISPLAY MOLECULAR GEOMETRY 
 list   : LIST OF CONTROL FILE 
 &      : GO BACK TO OCCUPATION/ORBITAL ASSIGNMENT MENU
 * or q : END OF DEFINE SESSION 



STATUS OF RI-OPTIONS:
    RI IS USED
    Memory for RI:           500 Mb
    Filename for auxbasis: auxbasis

 ENTER RI-OPTION TO BE MODIFIED 
      m: CHANGE MEMORY FOR RI 
      f: CHANGE FILENAME 
   jbas: ASSIGN AUXILIARY RI-J BASIS SETS
    off: TO SWITCH OFF RI 
 Use <ENTER>, q, end, or '*' to leave this menu


STATUS OF RI-OPTIONS:
    RI IS USED
    Memory for RI:           500 Mb
    Filename for auxbasis: auxbasis

 ENTER RI-OPTION TO BE MODIFIED 
      m: CHANGE MEMORY FOR RI 
      f: CHANGE FILENAME 
   jbas: ASSIGN AUXILIARY RI-J BASIS SETS
    off: TO SWITCH OFF RI 
 Use <ENTER>, q, end, or '*' to leave this menu
 GENERAL MENU : SELECT YOUR TOPIC 
 scf    : SELECT NON-DEFAULT SCF PARAMETER 
 mp2    : OPTIONS AND DATA GROUPS FOR rimp2 and mpgrad
 cc     : OPTIONS AND DATA GROUPS FOR ricc2
 pnocc  : OPTIONS AND DATA GROUPS FOR pnoccsd
 ex     : EXCITED STATE AND RESPONSE OPTIONS
 prop   : SELECT TOOLS FOR SCF-ORBITAL ANALYSIS 
 drv    : SELECT NON-DEFAULT INPUT PARAMETER FOR EVALUATION
          OF ANALYTICAL ENERGY DERIVATIVES 
          (GRADIENTS, FORCE CONSTANTS) 
 rex    : SELECT OPTIONS FOR GEOMETRY UPDATES USING RELAX
 stp    : SELECT NON-DEFAULT STRUCTURE OPTIMIZATION PARAMETER
 e      : DEFINE EXTERNAL ELECTROSTATIC FIELD 
 dft    : DFT Parameters 
 ri     : RI Parameters 
 rijk   : RI-JK-HF Parameters 
 rirpa  : RIRPA Parameters 
 gw     : OPTIONS AND DATA GROUPS FOR GW (escf) 
 senex  : seminumeric exchange parameters 
 hybno  : hybrid Noga/Diag parameters
 dsp    : DFT dispersion correction
 nmr    : NMR shift parameters
 ncoup  : NMR coupling parameters
 trunc  : USE TRUNCATED AUXBASIS DURING ITERATIONS 
 marij  : MULTIPOLE ACCELERATED RI-J 
 fde    : Frozen Density Embedding 

 dis    : DISPLAY MOLECULAR GEOMETRY 
 list   : LIST OF CONTROL FILE 
 &      : GO BACK TO OCCUPATION/ORBITAL ASSIGNMENT MENU
 * or q : END OF DEFINE SESSION 


 Reading data group $scfinstab from <control>


 MAIN MENU FOR RESPONSE CALCULATIONS

 OPTION | STATUS | DESCRIPTION
 -------------------------------------------------------------------
 rpas   | on     | RPA SINGLET EXCITATIONS (TDHF OR TDDFT)
 ciss   | off    | TDA SINGLET EXCITATIONS (CI SINGLES)
 rpat   | off    | RPA TRIPLET EXCITATIONS (TDHF OR TDDFT)
 cist   | off    | TDA TRIPLET EXCITATIONS (CI SINGLES)
 polly  | off    | STATIC POLARIZABILITY
 dynpol | off    | DYNAMIC POLARIZABILITY
 single | off    | SINGLET STABILITY ANALYSIS
 triple | off    | TRIPLET STABILITY ANALYSIS
 nonrel | off    | NON-REAL STABILITY ANALYSIS
 bse    | off    | BETHE-SALPETER EX.
 cbse   | off    | corr-aug. BETHE-SALPETER EX.

 ENTER <OPTION> TO SWITCH ON/OFF OPTION, * OR q TO QUIT



 MAIN MENU FOR RESPONSE CALCULATIONS

 OPTION | STATUS | DESCRIPTION
 -------------------------------------------------------------------
 rpas   | off    | RPA SINGLET EXCITATIONS (TDHF OR TDDFT)
 ciss   | off    | TDA SINGLET EXCITATIONS (CI SINGLES)
 rpat   | off    | RPA TRIPLET EXCITATIONS (TDHF OR TDDFT)
 cist   | off    | TDA TRIPLET EXCITATIONS (CI SINGLES)
 polly  | off    | STATIC POLARIZABILITY
 dynpol | off    | DYNAMIC POLARIZABILITY
 single | off    | SINGLET STABILITY ANALYSIS
 triple | off    | TRIPLET STABILITY ANALYSIS
 nonrel | off    | NON-REAL STABILITY ANALYSIS
 bse    | off    | BETHE-SALPETER EX.
 cbse   | off    | corr-aug. BETHE-SALPETER EX.

 ENTER <OPTION> TO SWITCH ON/OFF OPTION, * OR q TO QUIT

 GENERAL MENU : SELECT YOUR TOPIC 
 scf    : SELECT NON-DEFAULT SCF PARAMETER 
 mp2    : OPTIONS AND DATA GROUPS FOR rimp2 and mpgrad
 cc     : OPTIONS AND DATA GROUPS FOR ricc2
 pnocc  : OPTIONS AND DATA GROUPS FOR pnoccsd
 ex     : EXCITED STATE AND RESPONSE OPTIONS
 prop   : SELECT TOOLS FOR SCF-ORBITAL ANALYSIS 
 drv    : SELECT NON-DEFAULT INPUT PARAMETER FOR EVALUATION
          OF ANALYTICAL ENERGY DERIVATIVES 
          (GRADIENTS, FORCE CONSTANTS) 
 rex    : SELECT OPTIONS FOR GEOMETRY UPDATES USING RELAX
 stp    : SELECT NON-DEFAULT STRUCTURE OPTIMIZATION PARAMETER
 e      : DEFINE EXTERNAL ELECTROSTATIC FIELD 
 dft    : DFT Parameters 
 ri     : RI Parameters 
 rijk   : RI-JK-HF Parameters 
 rirpa  : RIRPA Parameters 
 gw     : OPTIONS AND DATA GROUPS FOR GW (escf) 
 senex  : seminumeric exchange parameters 
 hybno  : hybrid Noga/Diag parameters
 dsp    : DFT dispersion correction
 nmr    : NMR shift parameters
 ncoup  : NMR coupling parameters
 trunc  : USE TRUNCATED AUXBASIS DURING ITERATIONS 
 marij  : MULTIPOLE ACCELERATED RI-J 
 fde    : Frozen Density Embedding 

 dis    : DISPLAY MOLECULAR GEOMETRY 
 list   : LIST OF CONTROL FILE 
 &      : GO BACK TO OCCUPATION/ORBITAL ASSIGNMENT MENU
 * or q : END OF DEFINE SESSION 

 GENERAL MENU : SELECT YOUR TOPIC 
 scf    : SELECT NON-DEFAULT SCF PARAMETER 
 mp2    : OPTIONS AND DATA GROUPS FOR rimp2 and mpgrad
 cc     : OPTIONS AND DATA GROUPS FOR ricc2
 pnocc  : OPTIONS AND DATA GROUPS FOR pnoccsd
 ex     : EXCITED STATE AND RESPONSE OPTIONS
 prop   : SELECT TOOLS FOR SCF-ORBITAL ANALYSIS 
 drv    : SELECT NON-DEFAULT INPUT PARAMETER FOR EVALUATION
          OF ANALYTICAL ENERGY DERIVATIVES 
          (GRADIENTS, FORCE CONSTANTS) 
 rex    : SELECT OPTIONS FOR GEOMETRY UPDATES USING RELAX
 stp    : SELECT NON-DEFAULT STRUCTURE OPTIMIZATION PARAMETER
 e      : DEFINE EXTERNAL ELECTROSTATIC FIELD 
 dft    : DFT Parameters 
 ri     : RI Parameters 
 rijk   : RI-JK-HF Parameters 
 rirpa  : RIRPA Parameters 
 gw     : OPTIONS AND DATA GROUPS FOR GW (escf) 
 senex  : seminumeric exchange parameters 
 hybno  : hybrid Noga/Diag parameters
 dsp    : DFT dispersion correction
 nmr    : NMR shift parameters
 ncoup  : NMR coupling parameters
 trunc  : USE TRUNCATED AUXBASIS DURING ITERATIONS 
 marij  : MULTIPOLE ACCELERATED RI-J 
 fde    : Frozen Density Embedding 

 dis    : DISPLAY MOLECULAR GEOMETRY 
 list   : LIST OF CONTROL FILE 
 &      : GO BACK TO OCCUPATION/ORBITAL ASSIGNMENT MENU
 * or q : END OF DEFINE SESSION 


 ***********************************************************
 *                                                         *
 *                       e n d   o f                       *
 *                       D E F I N E                       *
 *                                                         *
 ***********************************************************


    ------------------------------------------------------------------------
         total  cpu-time :   0.08 seconds
         total wall-time :   0.73 seconds
    ------------------------------------------------------------------------

   ****  define : all done  ****


    2023-03-28 01:48:11.950 

